docs/key-issues/gpus.md

```diff
@@ -30,52 +30,66 @@ Packaging such projects for PyPI has been, and still is, quite challenging.
 
 ## Current state
 
-As of December 2022, PyPI and Python packaging tools are completely unaware of
+As of May 2023, PyPI and Python packaging tools are completely unaware of
 GPUs, and of CUDA. There is no way to mark a package as needing a GPU in sdist
 or wheel metadata, or as containing GPU-specific code (CUDA or otherwise). A
 GPU is hardware that may or may not be present in a machine that a Python
 package is being installed on - `pip` and other installers are unaware of this.
-If wheels contain CUDA code, they require CUDA Toolkit (a specific version of
-it at that) to be installed. Again, installers do not know this and there is no
-way to express this dependency. The same will be true for ROCm and other types
-of GPU hardware and languages.
-
-NVIDIA has made steps towards better support for CUDA on PyPI, through
-CUDA Python ([website](https://developer.nvidia.com/cuda-python), 
-[PyPI package](https://pypi.org/project/cuda-python)), however this is quite
-new and not used by other projects (nor does it target large projects like
-PyTorch and TensorFlow). For most of its own projects, it uses a
-[Private PyPI Index](https://pypi.org/project/nvidia-pyindex/) - and that also
-includes rebuilds of TensorFlow and other packages.
+If wheels contain CUDA code, they require the CUDA Toolkit (a specific version
+of it at that) to be installed. Again, installers do not know this and there is
```

Comment:
Not sure what "a specific version of it _at that_" refers to?

Reply:
It simply means there are version constraints, you must know if it's CUDA `11`, `12`, or even a more specific minor version like `11.1`. Maybe _"they require a specific version of the CUDA Toolkit"_ would be clearer?

Reply:
Perhaps there's just a lack of familiarity with this phrasing at play here? Does [this ref](https://idioms.thefreedictionary.com/at+that) help? In any case, happy to go with Ralf's rewording.